### ISSUE TITLE
Bug in models.py:2814 - get_file_path_with_link

### ISSUE THREAD

**Issue by joeppeeters** [DefectDojo/django-DefectDojo#6972]
Since [the last version](https://github.com/DefectDojo/django-DefectDojo/pull/6888) we encounter a bug in `models.py`.

It happens when we open the details of a finding.
`/findings/:finding_id`

Logs show the following
```
...
File "/app/./dojo/models.py", line 2814, in get_file_path_with_link
    return create_bleached_link(link, self.file_path)
UnboundLocalError: local variable 'link' referenced before assignment
```

**Comment by coheigea** [comment]
Do you have a redacted test file we could use to verify the fix?

**Comment by coheigea** [comment]
It's fixed in 2.15.1, this should be closed
